python/nav/web/auth.py

```diff
@@ -260,42 +260,54 @@ def login_remote_user(request):
     return None
 
 
+def ensure_account(request):
+    """Guarantee that request.account is set"""
+    session = request.session
+    account = getattr(request, 'account', None)
+
+    # Should only be the case before first page request
+    if ACCOUNT_ID_VAR not in session:
+        session[ACCOUNT_ID_VAR] = Account.DEFAULT_ACCOUNT
+
+    if account is None or account.id != session[ACCOUNT_ID_VAR]:
+        print('here 1', account, session)
```

Comment:
"Debugging"? I assume these print statements will be removed before the PR is completed :-)

Reply:
Ka-gone in next fixup-patch